### DISCUSSION TITLE
Broken link 'On-chip debugging'

### DISCUSSION THREAD

**Discussion by NageshEranki** [bitcraze/discussions#1135] [General]
I was following [instructions](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/openocd_gdb_debugging/#the-version-of-cortex-debug-tested-here-is-122) to use the debug adapter and under "Installing the SVD file", the [link](https://raw.githubusercontent.com/posborne/cmsis-svd/master/data/STMicro/STM32F405.svd) for the STM32F405.svd returns '404 not found'

**Comment by ataffanel** [comment]
This is something we have to fix in the documentation. In the mean-time, you can remove the line specifying this file in the configuration file and get going with debugging. This file is only required if you want to be able to see names of the microcontroller various peripherals registers in the debugger, so it is only useful for quite low level debugging.

**Reply by knmcguire** [reply]
I've made an issue of it here: https://github.com/bitcraze/crazyflie-firmware/issues/1342
